examples/examples.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python
```
This is the main entrypoint for running all examples with centralized configuration.
However, all examples are self0contained and can also be run directly.
```